src/main/java/Transcription/TranscriptionAdapter.java

```diff
@@ -0,0 +1,35 @@
+package Transcription;
+
+import Arguments.TranscriptArguments;
+import Search.Searchable;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class TranscriptionAdapter {
+    private List<String> files;
+    private VoskAdapter speechToText;
+
+    public TranscriptionAdapter(TranscriptArguments transcriptArguments) {
+        this.files = transcriptArguments.files;
+        this.speechToText = transcriptArguments.speechToText;
+    }
+
+    public List<Searchable> getSearchables() {
+        if (!files.isEmpty()) {
```

Comment:
Nitpick, but having this if statement around the logic of this function confuses what it does. It's also harder to read negative conditionals in general.  it would be better to move the edge case handling to the top, and then have the method body below:

`if (files.isEmpty()) {
 return new LinkedList<Searchable>();
}

// all the rest`

Reply:
Removed negative conditional and else statement.